services/evaluator/app/infrastructure/persistence/supabase/interview_repository.py

```
@@ -0,0 +1,118 @@
+        self.table_name = "interviews"
+    
+    async def create(self, interview: Interview) -> Interview:
+        """Create a new interview record in Supabase."""
+        try:
+            data = interview.to_dict()
+            result = self.supabase.table(self.table_name).insert(data).execute()
+            
+            if result.data:
+                return Interview.from_dict(result.data[0])
+            else:
+                raise Exception("Failed to create interview record")
```

Ruff (0.13.3)

Abstract `raise` to an inner function

(TRY301)